packages/inotify-tools/packaging

```diff
@@ -0,0 +1,17 @@
+set -eux
+
+sudo apt-get install -y autotools-dev automake libtool
```

Comment:
Yeah, the bosh philosophy seems to be packaging these dependencies--at least that's what we wound up doing in https://github.com/18F/cg-snort-boshrelease/blob/master/packages/snort/packaging, for example.